### DISCUSSION TITLE
Increase the distance

### DISCUSSION THREAD

**Discussion by MehmetOzkann** [micropython/micropython#12380] [nRF]
hi, i have nrf24l01 pa lna module and i am working with nrf24l01 library but in power and speed settings there is not enough resources for antenna module to increase distance. How can I adjust the dbm and speed settings according to the antenna module? The datasheet says it's up to 20 dBm for the transmitter and -95 dBm for the receiver.

**Comment by peterhinch** [comment]
That is an interesting module. The official driver operates the NRF24l01 at maximum power and minimum bandwidth. These are the conditions for maximum range. The PA/LNA part of the module is a purely electronic device with no way to configure its performance in code. As far as I can see all you can do is ensure you have a clear line of sight between transmitter and receiver and consider the use of directional antennas.

I do think the manufacturer's claim of 1100M may be optimistic. Have you considered Lora?

**Reply by MehmetOzkann** [reply]
I'm currently working on a project and we use NRF24l01 to give some basic datas from other rooms.Unfortunately the walls shorten the distance.I have worked with LoRa in electric car project and LoRa was more powerful than NRF24l01.Thanks for your answer.
